content/blog/getting-started-with-spark-on-mapr-sandbox.md

```diff
@@ -0,0 +1,217 @@
+---
+title: Getting Started with Spark on MapR Sandbox
+date: 2021-10-14T06:01:00.000Z
+author: MapR Tutorials
+authorimage: /img/Avatar1.svg
+---
+
+
+**Editor’s Note:** MapR products and solutions sold prior to the acquisition of such assets by Hewlett Packard Enterprise Company in 2019 may have older product names and model numbers that differ from current solutions. For information about current offerings, which are now part of HPE Ezmeral Data Fabric, please visit <https://www.hpe.com/us/en/software/ezmeral-data-fabric.html>
+
+At MapR, we distribute and support Apache Spark as part of the MapR Converged Data Platform. This tutorial will help you get started with running Spark applications on the MapR Sandbox.
+
+## Prerequisites
```

Comment:
Before stating what the Prerequisites and requirements are, this post really needs an introduction. You have to give the reader a reason to read further. Even if it is as simple as this: "This post is a tutorial designed to help you get started with Spark on the MapR Sandbox." But, it would be better if you could add a little something in on why they might want to use the MapR Sandbox and Spark here in the first place, maybe even relating back to a previous blog post that's been written.